newapi/debian/ooni_download_geoip.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
```

Comment:
This should generate logs and StatsD metrics so that we can alarm on it.
Also test the database at the end of the download.

Reply:
Done